### PR TITLE
Issue 5445: better documentation for cancel-url and one-time-login-url

### LINKED ISSUE #5445: Document that the `cancel-url` and `one-time-login-url` user tokens only work when sending out the admin notification emails
## Description of the bug

The tokens `cancel-url`  and `one-time-login-url` are listed under `['tokens']['user']` in the tokens array produced by `entity_get_tokens()`, yet these 2 tokens don't work.

## Steps To Reproduce
Try the following in Devel:

```php
$user = user_load(1);
$text = array(
  '[user:one-time-login-url]',
  '[user:cancel-url]',
);

foreach ($text as $t) {
  dpm(token_replace($t, array('user' => $user)), $t);
}
```

## Actual behavior
The output of the above shows that these two tokens are never replaced with the correct values.

## Expected behavior
The code above should output the one-time login URL and the account cancellation URL of the user.

## Additional information
The code for these replacements should have been included in `user_tokens()`. This should is a fairly simple fix.

### PR DESCRIPTION
**Pull request by argiepiano**
Fixes https://github.com/backdrop/backdrop-issues/issues/5445

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

File: core/modules/user/user.tokens.inc [hunk 1]
```diff
@@ -73,15 +73,21 @@ function user_token_info() {
     'description' => t('The URL of the account edit page.'),
     'type' => 'url',
   );
+
+  // This token is replaced only when the system sends an Account Cancellation email to a user.
```
Comment (klonos) [inline core/modules/user/user.tokens.inc]:
```suggestion
  // This token is replaced only when the system sends an Account Cancellation
  // email to a user.
```

File: core/modules/user/user.tokens.inc [hunk 2]
```diff
@@ -73,15 +73,21 @@ function user_token_info() {
...
+  // @see user_mail_tokens()
   $user['cancel-url'] = array(
     'name' => t('Account cancellation URL'),
-    'description' => t('The URL of the confirm delete page for the user account.'),
+    'description' => t('The URL of the confirm delete page for the user account. This token is only available when the system sends an Account Cancellation email to a user.'),
     'restricted' => TRUE,
     'type' => 'url',
   );
+
+  // This token is replaced only when the system sends a Password Recovery email to a user.
```
Comment (klonos) [inline core/modules/user/user.tokens.inc]:
```suggestion
  // This token is replaced only when the system sends a Password Recovery email
  // to a user.
```

**Comment by quicksketch** [comment]
Merged into 1.x and 1.20.x. Thanks!

File: core/modules/user/user.tokens.inc [hunk 3]
```diff
@@ -73,15 +73,23 @@ function user_token_info() {
...
     'type' => 'url',
   );
+
+  // This token is replaced only when the system sends an Account Cancellation
+  // email to a user.
+  // @see user_mail_tokens()
   $user['cancel-url'] = array(
     'name' => t('Account cancellation URL'),
-    'description' => t('The URL of the confirm delete page for the user account.'),
+    'description' => t('The URL of the confirm delete page for the user account. This token is only available when the system sends an Account Cancellation email to a user.'),
```
Comment (jenlampton) [inline core/modules/user/user.tokens.inc:82]: We should not be using the word "User". Let's shorten this to "This token is only available when the system sends an Account Cancellation email."

Comment (jenlampton) [reply core/modules/user/user.tokens.inc:82]: I've filed  a follow-up PR: https://github.com/backdrop/backdrop-issues/issues/5459
